en-US/snippets/2018/apr2018.lang

```diff
@@ -0,0 +1,59 @@
+## NOTE: Reference bug 1446192; Deadline: 9 April, 2018
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotions on a variety of topics
+
+
+# Campaign: Firefox Rocket
+# Link: https://app.adjust.com/2pv7s0
+;Satisfy your need for speed with <a>Firefox Rocket</a> — a lightweight and lightning fast browser for Android. Browse freely.
+Satisfy your need for speed with <a>Firefox Rocket</a> — a lightweight and lightning fast browser for Android. Browse freely.
+
+
+# Campaign: Firefox Rocket
+# Link: https://app.adjust.com/q96oqa
+;Get <a>Firefox Rocket for Android</a> — save on data costs and enjoy lightning fast browsing!
+Get <a>Firefox Rocket for Android</a> — save on data costs and enjoy lightning fast browsing!
+
+
+# Campaign: Firefox Rocket
+# Link: https://app.adjust.com/61qpmr
+;Power up Turbo Mode on <a>Firefox Rocket for Android</a> and browse even faster with a single click. Browse on.
+Power up Turbo Mode on <a>Firefox Rocket for Android</a> and browse even faster with a single click. Browse on.
+
+
+# Campaign theme: delight
+# This snippet suggests that connecting with nature makes people happy and sets them in a relaxed mood.
+;Watching the sunrise outdoors statistically increases your odds of having a good day. And needing a nap after lunch.
+Watching the sunrise outdoors statistically increases your odds of having a good day. And needing a nap after lunch.
+
+
+# Campaign: add-on
+# Link: https://developer.mozilla.org/Add-ons/WebExtensions/Your_first_WebExtension
+;Want to try building a browser extension? <a>Check out our tutorial</a>.
+Want to try building a browser extension? <a>Check out our tutorial</a>.
+
+
+# Campaign: Firefox Developer Edition
+# Link: https://www.mozilla.org/firefox/developer/
+;Are you a web developer? Try our fastest Firefox yet, now in Firefox Developer Edition. <a>Download now</a>.
+Are you a web developer? Try our fastest Firefox yet, now in Firefox Developer Edition. <a>Download now</a>.
+
+
+# Campaign: MDN
+# Link: https://developer.mozilla.org/docs/Games
+;Want to learn how to make games for the web? <a>MDN web docs</a> has you covered.
+Want to learn how to make games for the web? <a>MDN web docs</a> has you covered.
+
+
+# Campaign theme: delight
+# Link: https://youtu.be/FqnWEWYp9DU
+# The video sound track is in English. However, the localized Firefox browser will dictate the language the subtitles are in.
```

Comment:
The video will have subtitles if you manually enable them, and I don't think that's possible for a link. That's only possible if you embed the video in a page.

Honestly, I don't see much of a point in sending people to such video.